packages/venia-ui/lib/components/RichContent/richContent.js

```diff
@@ -3,6 +3,8 @@ import { useStyle } from '../../classify';
 import defaultClasses from './richContent.css';
 import { shape, string } from 'prop-types';
 import richContentRenderers from './richContentRenderers';
+import { setContentTypeConfig } from '@magento/pagebuilder/lib/config';
```

Comment:
We can not couple our theme with a standalone extension. From my point of view `pagebuilder` should have an extension point to inject local components into the page builder. There we need to mutch better API also maybe a Partner/Agency would overwrite the slider via this extension point.  Currently, it is not possible to register a target on page builder it self so we need to create a story in our backlog.